docs/sdk/src/reference_docs/chain_spec_runtime/tests/chain_spec_builder_tests.rs

```diff
@@ -88,9 +84,8 @@ fn generate_chain_spec() {
 		.arg("preset_2")
 		.output()
 		.expect("Failed to execute command");
-
+	println!("Expected: {:?}", output.stdout);
```

Comment:
reminder to remove once debugging is done

Reply:
Responding to this comment: https://github.com/paritytech/polkadot-sdk/pull/6624#issuecomment-2499728720. Is there anything present at WASM_FILE_PATH on your machine? Also, would be great to check the exit code, because it surely failed if the output is empty.

Did you try running the same cmd in terminal to see how it fails or if it works?

Reply:
@iulianbarbu , you were right: nothing in the `WASM_FILE_PATH` environment variable, and yes the output is empty.

Reply:
@iulianbarbu , the `chain-spec-guide-runtime/` folder is missing from the target folder.

Reply:
Can you do a `cargo build --release -p chain-spec-guide-runtime` and try again? Seems like tests depends on the runtime to be built prior to running it. Or even better, running a `cargo build --release -p polkadot-sdk-docs` should build everything that's needed most probably.

Reply:
@iulianbarbu , it worked!!!

Reply:
Additional question: shall I remove `Command::new(...)` wherever it is used, or only where `#[docify::export] ` is used?

Reply:
I think that wherever it is used should be fine, but mainly from the portions with `#[docify::export]`. BTW, would be great to extract the `cmd_lib` based commands in separate functions where we have `#[docify::export_content]`, leaving the assertion logic to a test which isn't exported in the docs. This example might help: https://github.com/paritytech/polkadot-sdk/blob/master/substrate/bin/utils/chain-spec-builder/tests/test.rs#L237.